pkg/strc/README.md

```diff
@@ -42,6 +42,16 @@ Spans end up in log sink too, for better readability, the following fields are a
 * `trace_id` - trace ID (disable by setting `strc.TraceIDFieldKey` to empty string)
 * `build_id` - build Git sha (disable by setting `strc.BuildIDFieldKey` to empty string)
 
+### Overriding time
+
+Span start, event and end time is automatically taken via `time.Now()` call but there are some use cases when this needs to be overridden to a specific time. Use special attributes to do that:
```

Comment:
Maybe an example of the use case would be good I think, can't immediately think of what the use case would be.

Reply:
Ah I see it's in the PR description, tbh I'd add it to the readme too, referencing the worker.